<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btNuevoCapitulo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNuevaResolucion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueva resolución</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPersona">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueva pesona</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNuevaCasa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nueva casa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeneral">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="actionCapitulos"/>
    <addaction name="actionNuevaPersona"/>
    <addaction name="actionNuevaCasa"/>
    <addaction name="actionNuevoCapitulo"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuGeneral"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCapitulos">
   <property name="text">
    <string>Capítulos</string>
   </property>
  </action>
  <action name="actionNuevoCapitulo">
   <property name="text">
    <string>Nuevo capítulo</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionNuevaCasa">
   <property name="text">
    <string>Nueva casa</string>
   </property>
  </action>
  <action name="actionNuevaPersona">
   <property name="text">
    <string>Nueva persona</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
